itests/org.openhab.core.ui.tests/src/main/java/org/eclipse/smarthome/ui/internal/items/ItemUIRegistryImplTest.java

```diff
@@ -56,12 +57,12 @@
  */
 public class ItemUIRegistryImplTest {
 
-    static private ItemUIRegistryImpl uiRegistry;
+    private static ItemUIRegistryImpl uiRegistry;

```

Comment:
I changed the order because that is the order we expect.
Regarding static or not, I don't know, I am not the guy who writes this part of the code but I will check.

Reply:
I can confirm the static was useless.